### PR TITLE
need to use archive virtualenv for venv setup

### PR DESCRIPTION
**Pull request by reedstrm**
- publishing is not deployed to db server

### CODE REVIEW

File: copy_database.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@
       environment:
         DB_URL: "postgresql://{{ archive_db_user }}:{{ archive_db_password }}@{{ archive_db_host }}:{{ archive_db_port }}/{{ db_name }}"
         DB_SUPER_URL: "postgresql://postgres:{{ archive_db_password }}@{{ archive_db_host }}:{{ archive_db_port }}/{{ db_name }}"
-      command: "/var/cnx/venvs/publishing/bin/cnx-db venv"
+      command: "/var/cnx/venvs/archive/bin/cnx-db venv"
```
Comment (karenc) [inline copy_database.yml]: Hm... is this meant to run locally on the database server?  Do we need to add `delegate_to: "{{ groups.database|first }}"`?

Comment (reedstrm) [reply copy_database.yml]: The underlying code will refuse to install the path-to-venv if it's not a local DB connection, ISTR.

Comment (reedstrm) [reply copy_database.yml]: BTW, this came about when Scott ran the db copy to staging. Production _has_ the proper path to the archive venv. Not sure how a path to a non-existent venv got installed.

Comment (karenc) [reply copy_database.yml]: Then we need the `delegate_to` line

**Pull request merged**